maxim/logger/openai/chat.py

```
@@ -8,10 +8,25 @@
 
 
 class MaximOpenAIChat(Chat):
+    """
+    This class represents a MaximOpenAIChat.
+    """
+
     def __init__(self, client: OpenAI, logger: Logger):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)